packages/headless/src/app/reducer-manager.ts

```diff
@@ -56,6 +57,12 @@ export function createReducerManager(
         .forEach((key) => (reducers[key] = newReducers[key]));
     },
 
+    remove(oldReducers: ReducersMapObject) {
+      Object.keys(oldReducers)
+        .filter((key) => key in reducers)
+        .forEach((key) => (reducers[key] = <Reducer>(() => {})));
```

Comment:
- Double iterations :x:
- Consider a `for..of` loop over a `forEach` to reduce the stack size.